internal/pkg/service/stream/mapping/table/column/renderer.go

```diff
@@ -86,3 +72,130 @@ func (r *Renderer) CSVValue(c Column, ctx recordctx.Context) (string, error) {
 
 	return "", errors.Errorf("unknown column type %T", c)
 }
+
+func (r *Renderer) mapPathCSVValue(c Path, ctx recordctx.Context) (string, error) {
+	bodyMap, err := ctx.BodyMap()
+	if err != nil {
+		return "", err
+	}
+
+	var value any
+
+	if c.Path == "" {
+		value = bodyMap
+	} else {
+		value, _, err = bodyMap.GetNested(c.Path)
+		if err != nil {
+			if c.DefaultValue != nil {
+				value = *c.DefaultValue
+			} else {
+				return "", errors.Wrapf(err, `path "%s" not found in the body`, c.Path)
+			}
+		}
+	}
+
+	if c.RawString {
+		if stringValue, ok := value.(string); ok {
+			return stringValue, nil
+		}
+	}
+
+	return json.EncodeString(value, false)
+}
+
+func (r *Renderer) jsonPathCSVValue(c Path, ctx recordctx.Context) (string, error) {
+	body, err := ctx.BodyBytes()
+	if err != nil {
+		return "", err
+	}
+
+	parser := r.fastjsonPool.Get()
+	defer r.fastjsonPool.Put(parser)
+
+	value, err := parser.ParseBytes(body)
```

Comment:
I would not do it if we would like to switch `fasthttp` to `net/http`. We'll be missing this method or? In case that we do not miss anything when switching just add it there.

Reply:
I don't think it's a small issue, the value must be cached, ... there can easily be 100 JSON columns.

Reply:
:+1: Yes, I saw it in the code.